scrapi/harvesters/biomedcentral.py

```diff
@@ -86,8 +86,15 @@ def schema(self):
             )
         }
 
-    def harvest(self, days_back=1):
-        search_url = self.URL.format(days_back)
+    def harvest(self, start_date=None, end_date=None):
+
+        start_date = start_date or (date.today() - timedelta(1))
```

Comment:
Right, agreed it should be consistent. I remember us agreeing that they should be datetimes as long as possible, but, in the base OAI harvester it made sense to turn them to strings earlier to account for the varying timezone granularity. 

Would it be too confusing to turn them to strings when going into the URL in most cases, but leave them as strings earlier in the base OAI harvester? Or should they all just become strings at the same time as the OAI harvester one does?